model/spectrum.h

```diff
@@ -0,0 +1,75 @@
+/*
+ * spectrum.h
+ */
+#ifndef DXTBX_MODEL_SPECTRUM_H
+#define DXTBX_MODEL_SPECTRUM_H
+
+#include <iostream>
+#include <cmath>
+#include <scitbx/vec3.h>
+#include <scitbx/array_family/shared.h>
+#include <scitbx/array_family/simple_io.h>
+#include <scitbx/array_family/simple_tiny_io.h>
+#include <dxtbx/error.h>
+#include "model_helpers.h"
+
+namespace dxtbx { namespace model {
+  typedef scitbx::af::shared<double> vecd;
+
+  /** A class to represent a 2D spectrum. */
+  class Spectrum {
+  public:
+    /** Default constructor: initialise all to zero */
+    Spectrum() {}
+
+    /**
+     * Initialise all the spectrum parameters.
+     * @param energies The spectrum energies (eV)
+     * @param energies The spectrum weights (unitless)
+     */
+    Spectrum(vecd energies, vecd weights)
+        : energies_(energies),
+          weights_(weights) {}
+
+    virtual ~Spectrum() {}
+
+    /* Get the spectrum energies (eV) */
+    vecd get_energies() const {
```

Comment:
Should the API be explicit that this is energies in eV not keV? i.e. `get_energies_eV()`?

Reply:
In my head, energy means eV and wavelength means angstrom.  I could see that that's not universal tho.  I'll make the change.